lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -843,14 +839,22 @@ function dosomething_campaign_is_active($node) {
    *
    * @param DateTime $date
    *  DateTime to edit
+   * @param boolean $add_four
+   *  Optional - prevents default EST datetime conversion from becoming previous day
    * @param boolean $end_of_day;
    *  Optional - set the date to end of day
    *
    * @return DateTime
    *  Updated time
    */
-function dosomething_campaign_set_est_timezone($date) {
+function dosomething_campaign_set_est_timezone($date, $add_four = FALSE, $end_of_day = FALSE) {
+  if ($add_four) {
+    $date->add(new DateInterval('PT4H'));
```

Comment:
why are we adding four? for UTC? not all users are in UTC so won't this break for people who have timezones set?